addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -168,27 +159,27 @@ def _cron_peppol_get_new_documents(self):
                             default_peppol_message_uuid=uuid,
                         )\
                         ._create_document_from_attachment(attachment.id)
-                    if partner_endpoint:
-                        move._message_log(body=_(
-                            'Peppol document has been received successfully. Sender endpoint: %s', partner_endpoint))
-                    else:
-                        move._message_log(
-                            body=_('Peppol document has been received successfully'))
                 # pylint: disable=broad-except
                 except Exception:
                     # if the invoice creation fails for any reason,
                     # we want to create an empty invoice with the attachment
-                    move = self.env['account.move'].create(error_move_vals)
+                    move = self.env['account.move'].create({
+                        'move_type': 'in_invoice',
+                        'peppol_move_state': 'done',
+                        'company_id': company.id,
+                        'extract_can_show_send_button': False,
+                        'peppol_message_uuid': uuid,
+                    })
                     attachment_vals.update({
                         'res_model': 'account.move',
                         'res_id': move.id,
                     })
                     self.env['ir.attachment'].create(attachment_vals)
-                    if partner_endpoint:
-                        move._message_log(body=_(
-                            'Failed to import a Peppol document. Sender endpoint: %s', partner_endpoint))
-                    else:
-                        move._message_log(body=_('Failed to import a Peppol document.'))
```

Comment:
Aren't these messages still relevant? 
Personally I still think it's important to inform the user that the import failed.

Reply:
@tsb-odoo

Reply:
tsb confirmed that we don't need to log that